src/Compilers/Lua/Portable/Syntax/Syntax.xml

```diff
@@ -1,1742 +1,1997 @@
+        <TypeComment>
+            A
+            <summary>Represents a tuple type binding</summary>
+        </TypeComment>
+        <FactoryComment>
+            <summary>
+                Creates a new <see cref="TupleTypeStnax" /> node.
+            </summary>
+        </FactoryComment>
+        <Kind Name="TypeBinding" />
+        <Field Name="OpenParenthesisToken" Type="SyntaxToken">
+            <PropertyComment>
+                <summary>The parameter list opening parenthesis.</summary>
+            </PropertyComment>
+            <Kind Name="OpenParenthesisToken" />
+        </Field>
+        <Field Name="Types" Type="SeparatedSyntaxList&lt;TypeSyntax&gt;">
+            <PropertyComment>
+                <summary>A list of types.</summary>
+            </PropertyComment>
+        </Field>
+        <Field Name="CloseParenthesisToken" Type="SyntaxToken">
+            <PropertyComment>
+                <summary>The close parenthesis token.</summary>
+            </PropertyComment>
+            <Kind Name="CloseParenthesisToken" />
+        </Field>
+    </Node>
+
+    <Node Name="TableTypeSyntax" Base="TypeSyntax">
+        <TypeComment>
+            <summary>Represents a table type binding</summary>
+        </TypeComment>
+        <FactoryComment>
+            <summary>
+                Creates a new <see cref="TableTypeSyntax" /> node.
+            </summary>
+        </FactoryComment>
+        <Kind Name="TypeBinding" />
+        <Field Name="ColonToken" Type="SyntaxToken" Optional="true" Override="true">
+            <PropertyComment>
+                <summary>The colon at the start of the statement (if any).</summary>
+            </PropertyComment>
+            <Kind Name="ColonToken" />
+        </Field>
+        <Field Name="Types" Type="SeparatedSyntaxList&lt;TableElementTypeSyntax&gt;">
+            <PropertyComment>
+                <summary>A list of types.</summary>
+            </PropertyComment>
+        </Field>
+    </Node>
+
+    <Node Name="VarargTypeSyntax" Base="TypeSyntax">
+        <TypeComment>
+            <summary>Represents a vararg type binding</summary>
+        </TypeComment>
+        <FactoryComment>
+            <summary>
+                Creates a new <see cref="VarargTypeSyntax" /> node.
+            </summary>
+        </FactoryComment>
+        <Kind Name="TypeBinding" />
+    </Node>
+
+    <Node Name="TypeCastSyntax" Base="ExpressionSyntax">
+        <TypeComment>
+            <summary>Represents a type cast</summary>
+        </TypeComment>
+        <FactoryComment>
+            <summary>
+                Creates a new <see cref="TypeCastSyntax" /> node.
+            </summary>
+        </FactoryComment>
+        <Kind Name="TypeCasting" />
+        <Field Name="Left" Type="ExpressionSyntax">
+            <PropertyComment>
+                <summary>The expression on the left side of the operator.</summary>
+            </PropertyComment>
+        </Field>
```

Comment:
I think a better name for this would be expression, as it's the only expression in the node
```suggestion
        <Field Name="Expression" Type="ExpressionSyntax">
            <PropertyComment>
                <summary>Gets the expression being cast to another type.</summary>
            </PropertyComment>
        </Field>
```